src/templates/en/2019/table_of_contents.html

```diff
@@ -213,10 +213,10 @@ <h2 class="part-name">Appendix</h2>
     </div>
 
     <section class="characters">
-      <img class="character" src="/static/images/character-painter.png" alt=""/>
-      <img class="character" src="/static/images/character-file.png" alt=""/>
-      <img class="character" src="/static/images/character-star.png" alt=""/>
-      <img class="character" src="/static/images/character-hat.png" alt=""/>
+      <img loading="lazy" class="character" src="/static/images/character-painter.png" alt=""/>
+      <img loading="lazy" class="character" src="/static/images/character-file.png" alt=""/>
+      <img loading="lazy" class="character" src="/static/images/character-star.png" alt=""/>
```

Comment:
Wouldn't bother lazy loading these:
```suggestion
      <img class="character" src="/static/images/character-star.png" alt=""/>
```